mtp_api/apps/mtp_auth/serializers.py

```diff
@@ -16,8 +16,8 @@
 
 
 class UserSerializer(serializers.ModelSerializer):
-    applications = serializers.SerializerMethodField()
-    permissions = serializers.SerializerMethodField()
+    applications = serializers.SerializerMethodField(read_only=True)
+    permissions = serializers.SerializerMethodField(read_only=True)
```

Comment:
yeah they are read-only, but i just wanted to make it a little clearer which fields are in fact used as POST parameters and which aren't. happy to remove if you like